docs/remote_inference_blueprints/cohere_connector_embedding_blueprint.md

```diff
@@ -0,0 +1,366 @@
+### Cohere Embedding Connector Blueprint:
+
+This blueprint will show you how to connect a Cohere embedding model to your Opensearch instance, including creating a k-nn index and your own Embedding pipeline. You will require a Cohere API key.
+
+Cohere currently offers the following Embedding models (model name and embedding dimensions):
+
+- embed-english-v3.0 1024
+- embed-multilingual-v3.0 1024
+- embed-english-light-v3.0 384
+- embed-multilingual-light-v3.0 384
+- embed-english-v2.0 4096
+- embed-english-light-v2.0 1024
+- embed-multilingual-v2.0 768
+
+See [Cohere's /embed API docs](https://docs.cohere.com/reference/embed) for more details.
+
+#### 1; Update your Opensearch cluster settings
+
+```json
+PUT /_cluster/settings
+
+{
+    "persistent": {
+        "plugins.ml_commons.trusted_connector_endpoints_regex": [
+          "^https://production\\.api\\.cohere\\ai/.*$",
+        ]
+    }
+}
+```
+
+If you have existing cluster settings, make sure to add them inside the array of endpoints.
+
+#### 2. Create your Model connector and Model group
+
+##### 2a. Register your Model group
+
+```json
+POST /_plugins/_ml/model_groups/_register
+
+{
+  "name": "cohere_model_group",
+  "description": "Your Cohere model group"
+}
+```
+
+This request response will return the `model_group_id`, note it down.
+
+##### 2b. Create your Model connector
+
+See above for all the values the `parameters > model` parameter can take.
+
+```json
+POST /_plugins/_ml/connectors/_create
+{
+  "name": "Cohere Embed Model",
+  "description": "The connector to Cohere's public embed API",
+  "version": "1",
+  "protocol": "http",
+  "credential": {
+    "cohere_key": "<ENTER_COHERE_API_KEY_HERE>"
+  },
+  "parameters": {
+    "model": "<ENTER_MODEL_NAME_HERE>",
```

Comment:
Add a hint here "choose your model from the provided model list"?

Reply:
Sounds good